client/app/components/toolbar_messageslist_actions.coffee

```diff
@@ -86,17 +82,12 @@ module.exports = ActionsToolbarMessagesList = React.createClass
 
             @props.afterAction() if @props.afterAction?
 
-            unless nextConversation.size
-                # Close 2nd panel : no next conversation found
-                @redirect @buildClosePanelUrl 'second'
-            else
-                # Goto to next conversation
-                @redirect
-                    direction: 'second',
-                    action: 'conversation',
-                    parameters:
-                        messageID: nextConversation.get('id')
-                        conversationID: nextConversation.get('conversationID')
+            # FIXME : move this into MessageActionCreator
+            # after deleting complete
+            # Goto to next conversation
+            window.location.href = RouterGetter.getURL
```

Comment:
FIXME : use `RouterActionCreator` to dispatch an action that will handle redirection.